src/index.js

```diff
@@ -15,26 +17,77 @@ const traverse = require('traverse');
 const DEFAULT_REPLACEMENT = '--REDACTED--';
 
 /**
- * Module exports.
+ * Validate serializers.
  */
 
-module.exports = (
+function validateSerializers(serializers) {
+  serializers.map(({ path, serializer }) => {
```

Comment:
Good point. I think there is no need for that since the lodash `get` already does some validation internally. If the path isn't valid, the `get` before updating will return `undefined` and skip that value.